cpp/include/raft/neighbors/detail/knn_brute_force.cuh

```diff
@@ -265,7 +285,8 @@ void brute_force_knn_impl(
   bool rowMajorQuery                  = true,
   std::vector<IdxType>* translations  = nullptr,
   raft::distance::DistanceType metric = raft::distance::DistanceType::L2Expanded,
-  float metricArg                     = 0)
+  float metricArg                     = 0,
```

Comment:
Can we default this to 2.0 everywhere? That's usually what I see in scikit learn and other places as well. It's only used in Minkowski/lp currently, which would make it equivalent to euclidean.